web/filter/jwt/jwt_test.go

```diff
@@ -78,12 +78,12 @@ func TestVerifyJWT_no_JWT(t *testing.T) {
 
 	name, err := GetName(r.Context())
 	require.Equal(t, "", name)
-	require.EqualError(t, err, "failed to get name: no user in context")
-	require.EqualError(t, errors.Unwrap(err), "no user in context")
-	isAdmin, err := IsAdmin(r.Context())
+	require.EqualError(t, err, "failed to get name: failed to get token claims: no token in context")
+	require.EqualError(t, errors.Unwrap(err), "no token in context")
+	isAdmin, err := HasRole(r.Context(), "staff")
 	require.False(t, isAdmin)
-	require.EqualError(t, err, "failed to get admin status: no user in context")
-	require.EqualError(t, errors.Unwrap(err), "no user in context")
+	require.EqualError(t, err, "failed to check for 'staff' role: failed to get token claims: no token in context")
+	require.EqualError(t, errors.Unwrap(err), "no token in context")
 }
 
 func TestVerifyJWT_valid_JWT_admin(t *testing.T) {
```

Comment:
Tests (as well as documentation) should use RFC6761 domains (e.g. `example.com` or `whatever.test`, depending on whether `NXDOMAIN` responses are acceptable in the respective cases).

Reply:
the domain in the token is never resolved, so tentatively: won't do